src/bin/julialauncher.rs

```diff
@@ -5,9 +5,18 @@ use juliaup::config_file::{load_config_db, JuliaupConfig, JuliaupConfigChannel};
 use juliaup::global_paths::get_paths;
 use juliaup::jsonstructs_versionsdb::JuliaupVersionDB;
 use juliaup::versions_file::load_versions_db;
+#[cfg(not(windows))]
+use nix::{
+    sys::wait::{waitpid, WaitStatus},
+    unistd::{fork, ForkResult},
+};
 use normpath::PathExt;
+#[cfg(not(windows))]
+use std::os::unix::process::CommandExt;
 use std::path::Path;
 use std::path::PathBuf;
+#[cfg(windows)]
+use std::process::Command;
```

Comment:
There are some GH Action warning about  "unused import: `std::process::Command`"

I also noticed that the "not(windows)" code path also seems to be using `std::process::Command` (well, `std::process::Command::new`)... So maybe this is just not needed? Anyway, I don't know Rust, so I may have just said something incredibly idiotic, my apologies if I am wasting your time :-(

Reply:
Apparently safe to remove.